sonaranalyzer-dotnet/its/expected/Ember-MM/Ember.Plugins-{9496C697-5AFD-4813-AEDC-AF33FACEADF0}-S2221.json

```diff
@@ -0,0 +1,17 @@
+{
+"issues":  [
+{
+"id":  "S2221",
+"message":  "Catch a list of specific exception subtype or use exception filters instead.",
+"location":  {
+"uri":  "Ember-MM\Ember.Plugins\PluginSectionHandler.cs",
```

Comment:
One of the tests raises on this :)

try
{
    (code)
}
catch
{
    //TODO: Hmmm?
}